src/pages/trader-tools/common/_style.js

```diff
@@ -16,26 +16,40 @@ export const Hero = styled(Flex)`
         height: 210px;
     }
 `
+export const BreadCrumbContainer = styled(Container)`
+    margin-top: 2.4rem;
+
+    @media ${device.laptopL} {
+        width: 100%;
+    }
+    @media ${device.laptopM} {
+        width: 90%;
+    }
+`
 
 export const StyledSection = styled(SectionContainer)`
     position: relative;
-    padding: 3rem 0;
+    padding: 8rem 0;
+
+    @media ${device.laptopM} {
+        padding: 40px 0;
+    }
 `
 
-export const SectionSubtitle = styled(Text)`
+export const SectionSubtitle = styled(Header)`
     width: 79.2rem;
-    margin: auto;
-    margin-bottom: 4rem;
+    margin: 0 auto;
+    line-height: 1.5;
```

Comment:
isn't lineheight its set on header?

Reply:
Yeah it is but this is due to the fact that design isn't the same as our `typography` configuration.